skiko/docker/linux-amd64/Dockerfile

```diff
@@ -10,14 +10,17 @@ RUN apt-get update -y && \
     apt-get update && \
     apt-get install gcc-9 g++-9 -y && \
     update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 60 --slave /usr/bin/g++ g++ /usr/bin/g++-9 && \
-    update-alternatives --config gcc 
+    update-alternatives --config gcc \
```

Comment:
To build a snapshot of skiko on Teamcity from this PR,  we would need the new docker image sooner than this PR is merged.

Would you like to make a separate PR with docker change?

Reply:
I'll build a new docker image now.

But I see that our Teamcity build configuration will need to have an extra step added to build for linuxArm64 - it won't build and publish automatically. 

The build configuration is not open sourced unfortunately. So I'll need to make that change. I'll try to do that by the end of this week.

Reply:
And to make the new CI step work we actually need your PR to merged before it runs. It means we can't have a snapshot build of skiko for your PR.

I'll test your changes locally on my linux machine tomorrow.